avd-containers/dev/.devcontainer/entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+AVD_DEV_REQ_FILE="${HOME}/.ansible/collections/ansible_collections/arista/avd/requirements-dev.txt"
+USERNAME=$(whoami)
+
+# Install ansible if not yet installed and avd collection is locally mounted.
+ansible --version &> /dev/null ||  if [ -f $AVD_DEV_REQ_FILE ]; then
+  sudo chown -R ${USERNAME} ${HOME}/.ansible
+  ANSIBLE_CORE_VERSION=$(cat ${AVD_DEV_REQ_FILE}| grep ansible-core)
+  pip3 install "${ANSIBLE_CORE_VERSION}"
+  ansible-galaxy collection install -r ${HOME}/.ansible/collections/ansible_collections/arista/avd/collections.yml
+  pip3 install -r ${HOME}/.ansible/collections/ansible_collections/arista/avd/requirements.txt
+  pip3 install -r ${HOME}/.ansible/collections/ansible_collections/arista/avd/requirements-dev.txt
+fi
```

Comment:
Change this to use paths relative from the entrypoint.sh file itself, so we operate from the workspace files instead of the .ansible path.